src/store/authors/authors.js

```diff
@@ -0,0 +1,44 @@
+import { createSlice, createAsyncThunk } from '@reduxjs/toolkit';
+import { apiService } from '../../sesrvice';
+
+const initialState = {
+	authors: [],
+};
+
+export const fetchAllAuthors = createAsyncThunk(
```

Comment:
Please move it to the separate file "actions.ts"

Reply:
move it to the separate file "actions.ts"